report/Evaluation/Essence.tex

```diff
@@ -0,0 +1,56 @@
+\section{Essence}\label{sec:eval_essence}
+In this section, we will reflect on Essence, and talk about some of the things that went well-using Essence, and some things that could be better, or changed for a better understanding of it.
+
+\subsection{Ecology}
+We had some problems understanding what External services and External artifacts are, what they could be, and what the difference between them is.
+This lead to some discussions that could have been avoided if we had a better understanding of it, and the lines between them where clearer.
+
+An idea we had, to this work process, is that before you classify the different points, you should just create a list with all of the points that could be used in this project.
+After the creation of the list, then classify the different points into Ecology terms.
+Because we had the problem with External services and artifacts, we thought that these two points could be merged into one, which still captures the meaning of these two points.
+
+Regarding the SWOT method, it gave meaning to use this method in Ecology. 
+It is important to know if we are using the right people or services.
+One thing that helped us, was that we had written down a small piece of text to each element, in regards to SWOT, and this made it easy for us to score them, because we could, rather fast, see what the pros and cons were in a given element.
+If we had not done this, this process might have been more challenging, because we needed to remember what was the pros and cons, and we needed to be on the same page, in terms of our understanding of the element in question.
+
+Overall we liked the process in Ecology, and it was fun to give the different elements points, in regards to the SWOT method. 
+Here we had some good discussions on why we though a given element, had the strength it had and so forth.
+% \\TODO Find out if this sentense needs to be here or not.
+As we were done with this process, we came to the conclusion that it will be easier to understand the difference between the boxes, and therefore you can identify points faster, because you have this understanding of Essence, after utilizing Essence.
```

Comment:
I am not sure what you are trying to say here ?

Reply:
Forgot that i had a TODO there. 
Will remove the sentense, as it does not give anything, and is not formulated correctly.
```suggestion
```